### PR TITLE
Adding xpath function to Gravitee EL plugin

### PR DESCRIPTION
**Pull request by faraday-m**
Pull request for gravitee-io/issues#3211

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -106,5 +111,27 @@
         <gravitee-common.version>1.15.0</gravitee-common.version>
         <jmh.version>1.21</jmh.version>
     </properties>
-
+    <build>
```
Comment (brasseld) [inline pom.xml]: This part is not required

File: src/main/assembly/plugin-assembly.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (brasseld) [inline src/main/assembly/plugin-assembly.xml]: This file is not required

**Comment by brasseld** [comment]
@faraday-m LGTM

Please squash the 2 commits into a single one.

Thanks

**Comment by faraday-m** [comment]
> I'm a bit nervous about all the transitive dependencies we get by adding the spring-integration-xml dependency:
> 
> ```
> +- org.springframework.integration:spring-integration-xml:jar:5.1.5.RELEASE:compile
> [INFO] |  +- org.springframework.integration:spring-integration-core:jar:5.1.5.RELEASE:compile
> [INFO] |  |  +- org.springframework:spring-messaging:jar:5.1.7.RELEASE:compile
> [INFO] |  |  +- org.springframework:spring-tx:jar:5.1.7.RELEASE:compile
> [INFO] |  |  +- org.springframework.retry:spring-retry:jar:1.2.4.RELEASE:compile
> [INFO] |  |  \- io.projectreactor:reactor-core:jar:3.2.9.RELEASE:compile
> [INFO] |  |     \- org.reactivestreams:reactive-streams:jar:1.0.2:compile
> [INFO] |  +- org.springframework:spring-oxm:jar:5.1.7.RELEASE:compile
> [INFO] |  +- org.springframework.ws:spring-xml:jar:3.0.7.RELEASE:compile
> [INFO] |  |  \- commons-io:commons-io:jar:2.5:compile
> [INFO] |  \- org.springframework.ws:spring-ws-core:jar:3.0.7.RELEASE:compile
> [INFO] |     +- org.springframework:spring-web:jar:5.0.10.RELEASE:compile
> [INFO] |     \- org.springframework:spring-webmvc:jar:5.0.10.RELEASE:compile
> ```
> 
> Are you sure we need all of them?

I added some exclusions in pom.xml, looks like it works

**Comment by faraday-m** [comment]
@brasseld is everything ok with dependencies now?

**Comment by faraday-m** [comment]
Up

**Pull request closed without merging**